app/assets/javascripts/components/students/shared/StudentList/Student/Student.jsx

```diff
@@ -131,6 +131,24 @@ const Student = createReactClass({
       );
     }
 
+    let contentAdded;
+    if (course.home_wiki_bytes_per_word) {
+      const mainspaceWords = Math.round(student.character_sum_ms / course.home_wiki_bytes_per_word);
+      const userspaceWords = Math.round(student.character_sum_us / course.home_wiki_bytes_per_word);
+      const draftWords = Math.round(student.character_sum_draft / course.home_wiki_bytes_per_word);
```

Comment:
Maybe we can make a utility class / file that encapsulates all of this math? That way it can be used elsewhere more easily.